Corona-Warn-App/src/test/java/de/rki/coronawarnapp/nearby/windows/ExposureWindowsCalculationTest.kt

```diff
@@ -0,0 +1,405 @@
+        result.append("\n\t").append("↳ Report Type Offset Self Report: ${config.transmissionRiskLevelEncoding.reportTypeOffsetSelfReport}")
+
+        result.append("\n").append("◦ Transmission Risk Level Filters (${config.transmissionRiskLevelFilters.size})")
+        for (filter: RiskCalculationParametersOuterClass.TrlFilter in config.transmissionRiskLevelFilters) {
+            result.append("\n\t").append("⇥ Trl Filter")
+            result.append(logRange(filter.dropIfTrlInRange, "Drop If Trl In Range"))
+        }
+
+        result.append("\n").append("◦ Transmission Risk Level Multiplier: ${config.transmissionRiskLevelMultiplier}")
+        result.append("\n").append("-------------------------------------------- ⚙ -").append("\n")
+        debugLog(result.toString(), LogLevel.NONE)
+    }
+
+    private fun logRange(range: RiskCalculationParametersOuterClass.Range, rangeName: String): String {
+        val builder = StringBuilder()
+        builder.append("\n\t\t").append("⇥ $rangeName")
+        builder.append("\n\t\t\t").append("↳ Min: ${range.min}")
+        builder.append("\n\t\t\t").append("↳ Max: ${range.max}")
+        builder.append("\n\t\t\t").append("↳ Min Exclusive: ${range.minExclusive}")
+        builder.append("\n\t\t\t").append("↳ Max Exclusive: ${range.maxExclusive}")
+        return builder.toString()
+    }
+
+    private fun logExposureWindow(exposureWindow: ExposureWindow, title: String, logLevel: LogLevel = LogLevel.ALL) {
+        val result = StringBuilder()
+        result.append("\n\n").append("------------ $title -----------")
+        result.append("\n").append("Mocked Exposure window: #${exposureWindow.hashCode()}")
+        result.append("\n").append("◦ Calibration Confidence: ${exposureWindow.calibrationConfidence}")
+        result.append("\n").append("◦ Date Millis Since Epoch: ${exposureWindow.dateMillisSinceEpoch}")
+        result.append("\n").append("◦ Infectiousness: ${exposureWindow.infectiousness}")
+        result.append("\n").append("◦ Report type: ${exposureWindow.reportType}")
+
+        result.append("\n").append("‣ Scan Instances (${exposureWindow.scanInstances.size}):")
+        for (scan: ScanInstance in exposureWindow.scanInstances) {
+            result.append("\n\t").append("⇥ Mocked Scan Instance: #${scan.hashCode()}")
+            result.append("\n\t\t").append("↳ Min Attenuation: ${scan.minAttenuationDb}")
+            result.append("\n\t\t").append("↳ Seconds Since Last Scan: ${scan.secondsSinceLastScan}")
+            result.append("\n\t\t").append("↳ Typical Attenuation: ${scan.typicalAttenuationDb}")
+        }
+        result.append("\n").append("-------------------------------------------- ✂ ----").append("\n")
+        debugLog(result.toString(), logLevel)
+    }
+
+    private fun jsonToConfiguration(json: DefaultRiskCalculationConfiguration) {
```

Comment:
Can we call this `setupTestConfiguration`? or have this return a config, not set it, itself.

Reply:
done